### PR TITLE
Update menu_pl.lang

### PR DESCRIPTION
**Pull request by Halamix2**
I had to do this manually since Weblate added only 7 strings

### CODE REVIEW

**Comment by Cervator** [comment]
Yeah that's weird. Weblate sees all 132 strings just fine but seemingly only updated the existing ones on GitHub?

Interestingly the "Download for an offline translation" link in Weblate also only catches the 7 strings version of the file. Wonder if it is a bug fixed in a later version.

@nh-99 this is why we need your Ansible-powered antics to save our translations! :D

**Comment by GooeyHub** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.terasology.org/job/TerasologyPRs/318/
Hooray Jenkins reported success with all tests good!

File: engine/src/main/resources/assets/i18n/menu_pl.lang [hunk 1]
```diff
@@ -1,9 +1,134 @@
...
+    "create-preview": "Podgląd",
+    "create-game-title": "Utwórz grę",
+    "game-name": "Nazwa gry",
+    "gameplay": "Szablon rozgrywki",
+    "gameplay-info": "można kliknąć \"moduły\" aby dostosować",
+    "select-modules": "Moduły...",
+    "world-config-preview": "Szczegóły...",
+    "ingame-menu-title": "Menu w grze",
+    "select-world-gen": "Wybierz generator świata",
+    "select-world-gen-info": "Włączenie modułów może dodać ich więcej",
```
Comment (Halamix2) [inline engine/src/main/resources/assets/i18n/menu_pl.lang]: I hate own typos

**Comment by GooeyHub** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.terasology.org/job/TerasologyPRs/319/
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Weird, I merged this but it didn't auto-close as merged. It is present in develop now though.

Submitted #2029 for the issue where some special characters do not show.

@Halamix2 / @msteiger / @skaldarnar should we go ahead and update the weblate branch whenever we do a merge like this?

**Comment by skaldarnar** [comment]
What do you mean with "update the weblate branch"?

**Comment by Cervator** [comment]
We merged to develop including changes from the weblate branch. I figured we should then push from develop to weblate to have it fully up to date again :-)

**Pull request closed without merging**